### PR TITLE
Refactor satellite notebook

### PR DESCRIPTION
**Pull request by jrleeman**
Refactor and clean up exercises.

### CODE REVIEW

File: notebooks/Satellite_Data/PlottingSatelliteData.ipynb [hunk 1]
```diff
@@ -498,16 +516,39 @@
...
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we have two timestamps! Let's remove the old one (this is why we kept a handle to it)."
+    "### Using Colortables\n",
+    "The map is much improved now, but it would look much better with a different color scheme. \n",
+    "\n",
+    "Colormapping in matplotlib (which backs CartoPy) is handled through two pieces:\n",
+    "\n",
+    "- The colormap controls how values are converted from floating point values in the range [0, 1] to colors (think colortable)\n",
```
Comment (dopplershift) [inline notebooks/Satellite_Data/PlottingSatelliteData.ipynb]: If you flip the order of these bullets, they would read in the order of how the steps are actually applied to convert from their original data to an image. I'm not sure if you had another intention in mind here behind the ordering, though.

File: notebooks/Satellite_Data/PlottingSatelliteData.ipynb [hunk 2]
```diff
@@ -195,42 +225,37 @@
...
-    "print(ds['Sectorized_CMI'])"
+    "ds = ds.remote_access(use_xarray=True)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This reveals several useful pieces of information, but we're going to focus on the `grid_mapping` data. This attribute is defined by the [NetCDF Climate and Forecast (CF) Metadata Conventions](http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html) and specifies a variable that contains information about the grid's projection."
+    "Yes, we reused the `ds` name - mostly just to help keep our namespace realtively uncluttered, but no need to. This is one of the powers of Python - `ds` can be a string, a class, an integer, or anything else all during the course of one program."
```
[the remark this answers is not in the document]
Comment (jrleeman) [reply notebooks/Satellite_Data/PlottingSatelliteData.ipynb]: I was following my predecessors pattern 👿 - I'm open to changing it certainly, but is that useful to keep the old handle around?

**Comment by jrleeman** [comment]
Revised per your suggestions @dopplershift

File: notebooks/Satellite_Data/PlottingSatelliteData.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (dopplershift) [reply notebooks/Satellite_Data/PlottingSatelliteData.ipynb]: Yeah, your predecessor made mistakes. 😉 I don’t think we need to keep the old handle around, but I don’t think we should be demonstrating re-using the variable name. We wouldn’t want that in MetPy code and it also leads to confusing behavior in the notebook—it makes re-running cells problematic.

[the remark this answers is not in the document]
Comment (jrleeman) [reply notebooks/Satellite_Data/PlottingSatelliteData.ipynb]: Addressed

**Comment by dopplershift** [comment]
Travis failure looks real.

**Pull request merged**